Loop/Managers/LoopDataManager.swift

```diff
@@ -1006,11 +1006,11 @@ extension LoopDataManager {
         var retrospectiveGlucoseEffect = self.retrospectiveGlucoseEffect
         var effects: [[GlucoseEffect]] = []
 
+        let insulinCounteractionEffects = insulinCounteractionEffectsOverride ?? self.insulinCounteractionEffects
```

Comment:
Pulled up since the potential bolus uses the last effect date to filter its date range; missed this before.

Reply:
Would this explain the to-high recommendation and low prediction that Matt saw? If not, do you have any ideas about what might've caused what he saw?

Reply:
This change affects the manual glucose entry code path, but it looked like Matt's case didn't involve manual entry. Will ping you in Slack with more thoughts on what he saw.